sites/partners/.babelrc

```diff
@@ -1,4 +0,0 @@
-{
-  "presets": ["next/babel"],
```

Comment:
@ifranch the Babel adds in https://github.com/metrotranscom/doorway/commit/9fb370c23dfb8bf15832ad0bd487b0c58f4fa4d1 are conflicting heavily with Bloom main -- could you give me some context for what Babel is used for (also happy to pair and resolve conflicts together)